File: Week_7/problem_3-Square_or_rectangle.py
# https://codeforces.com/group/MWSDmqGsZm/contest/329103/problem/A

def square_or_rectangle_checker(a, b):
    if a == b:
        return "Square"
    else:
        return "Rectangle"

n = int(input())

results = []  # To store results here

for _ in range(n):
    a, b = map(int, input().split())  # Read the pair of numbers
    results.append(square_or_rectangle_checker(a, b))  # Add result to the list

# After all inputs are read, print all results
for result in results:
    print(result)

File: Week_7/problem_3-Square_or_rectangle.c
// https://codeforces.com/group/MWSDmqGsZm/contest/329103/problem/A

#include <stdio.h>
#include <limits.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

void square_or_rectangle_checker(int a, int b)
{
    if (a == b)
    {
        printf("Square\n");
    }
    else
    {
        printf("Rectangle\n");
    }
}

int main()
{
    int n;
    scanf("%d", &n);

    for (int i = 0; i < n; i++)
    {
        int a, b;
        scanf("%d %d", &a, &b);
        square_or_rectangle_checker(a, b);
    }

    return 0;
}
